avro-data/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -2156,7 +2156,7 @@ private Object defaultValueFromAvroWithoutLogical(Schema schema,
 
       case RECORD: {
         if (!jsonValue.isObject()) {
-          throw new DataException("Invalid JSON for record default value: " + jsonValue.toString());
+          throw new DataException("Invalid JSON for record default value: <redacted>");
```

Comment:
@rayokota what does default value signify here?
Is it good to truncate this to "Invalid JSON for record"